plugin/build_baron.go

```diff
@@ -36,7 +37,19 @@ func (bbp *BuildBaronPlugin) Configure(conf map[string]interface{}) error {
 	}
 
 	for projName, proj := range bbpOptions.Projects {
-		webhookConfigured := proj.TaskAnnotationSettings.FileTicketWebHook.Endpoint != ""
+		webHook := proj.TaskAnnotationSettings.FileTicketWebHook
+		flags, err := evergreen.GetServiceFlags()
+		if err != nil {
+			return errors.Wrap(err, "error getting service flags")
+		}
+		if flags.PluginAdminPageDisabled {
+			wh, ok := IsWebhookConfigured(projName, "")
+			if !ok {
```

Comment:
I don't think we should check this here; we already check if there's an error at line 53 and 56 but it's not actually an error otherwise (also we're wrapping an error that doesn't exisst)